.github/workflows/example-local.yml

```diff
@@ -38,4 +39,5 @@ jobs:
       - name: Upload provenance
         uses: actions/upload-artifact@v2
         with:
+          name: build.provenance
```

Comment:
Should we update this to use the in-toto bundle naming? https://github.com/in-toto/attestation/blob/main/spec/bundle.md

Reply:
Not in this PR, since it's not in JSON Lines format (it's currently pretty-printed).

Reply:
Also it's unsigned so it's not a DSSE envelope.

Reply:
Ah, got it.  Two very important points.